### ISSUE TITLE
Allow AMP viewers to suppress story info button/dialog with a URL parameter

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#23574]
Currently, the [info button and dialog](https://github.com/ampproject/amphtml/issues/14923) ("(i) icon") are shown whenever stories are embedded in any viewer, however some viewers may wish to opt-out of this feature.  Since there are already viewers live that are depending on this, but no such API to control the feature exists, I propose we add an optional URL parameter with the following semantics:

**`showstoryurlinfo`**:
- `auto` [default]: has the same behavior as `always` when in a viewer, and the same behavior as `never` when not in a viewer
- `always`: always shows the story URL info button and dialog, as long as this parameter is present
- `never`: never shows the story URL info button and dialog, as long as this parameter is present

**Comment by newmuis** [comment]
We're having additional conversations around this, which might affect the API design.  The "(i) icon" experience itself is going to be changing to better accommodate platforms that allow swiping between stories, and I want to make sure we have solidified plans for what that experience will look like before making API design choices.  While it's still up in the air, I think the API I proposed above is not what we would go with; I hope to provide more concrete details by next week.
